### PR TITLE
added src files

### PR DESCRIPTION
**Pull request by ivanboika**
Modules and src files

### CODE REVIEW

File: src/Connection.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import psycopg2
+
+
+class DAO:
+    def __init__(self):
+        self.conn = psycopg2.connect("dbname=app user=postgres password=1221909128 host=localhost")
```
Comment (Orange-hanter) [inline src/Connection.py]: move all credentials to separate `Config.py`. Moreover, it should have logic of an alternative initialization via `system environment` (last one to investigate but not necessary to implementation)

File: src/MainWindow.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                    pixmap.loadFromData(field)
+                    photos.append(pixmap)
+
+        model = TableModel(data, self.columns, photos)
+        self.table.setModel(model)
+
+        self.table.resizeRowsToContents()
+        self.table.resizeColumnsToContents()
+
+        self.table.resize(600, 400)
```
Comment (Orange-hanter) [inline src/MainWindow.py]: make global variables

File: src/MainWindow.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            self.tableChoice.addItem(tableName[0])
+
+        self.setTable()
+
+    def itemChanged(self):
+        self.setTable()
+
+    def addButtonClicked(self):
+        dialogWin = QDialog()
+        dialogWin.setWindowTitle('Adding window')
```
Comment (Orange-hanter) [inline src/MainWindow.py]: such way isn't good. We need investigate about localization. All work we do is for russian speakers

**Review by Orange-hanter — CHANGES_REQUESTED** [review]

File: src/Connection.py (hunk 1 above)
Comment (ivanboika) [reply src/Connection.py]: Yup, I'll do it asap

File: src/MainWindow.py (hunk 3 above)
Comment (ivanboika) [reply src/MainWindow.py]: So, should I change all the to ru language?

File: src/MainWindow.py (hunk 2 above)
Comment (ivanboika) [reply src/MainWindow.py]: I'm not sure about this line, probably this one will not exist soon
Rn I can't make this table auto-resizable only by table's internal data(
But I understand your point

File: src/MainWindow.py (hunk 3 above)
Comment (ivanboika) [reply src/MainWindow.py]: Did it

File: src/MainWindow.py (hunk 2 above)
Comment (ivanboika) [reply src/MainWindow.py]: I made a table kinda auto-resizable, so it's commented rn

File: src/Connection.py (hunk 1 above)
Comment (ivanboika) [reply src/Connection.py]: Done

File: src/starting.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from src.MainWindow import *
```
Comment (Orange-hanter) [inline src/starting.py]: Try to avoid importing all

File: src/MainWindow.py [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+        buttonLayout.addWidget(QPushButton('Поиск записи'), 2, 0)
+        buttonLayout.addWidget(self.table, 0, 1, 8, 8, Qt.AlignTop)
+        #buttonLayout.setAlignment(Qt.AlignLeft)
+
+        self.setLayout(buttonLayout)
+
+        #self.widgetLayout.setLayout(buttonLayout)
+        #self.widgetLayout.adjustSize()
+
+        self.dao.exec("""SELECT table_name FROM information_schema.tables
```
Comment (Orange-hanter) [inline src/MainWindow.py]: SRP violation. Try to not mix UI logic and business logic. At least here could be nice idea to use some MVC or MVVM pattern. I suppose it better to discuss it.

File: src/MainWindow.py (hunk 3 above)
Comment (Orange-hanter) [reply src/MainWindow.py]: Hmmm, I found interesting stuff
https://ru.stackoverflow.com/questions/521034/Способы-мультиязычности-программ-на-python
Look please

Comment (ivanboika) [reply src/MainWindow.py]: Yup

File: src/starting.py (hunk 4 above)
Comment (ivanboika) [reply src/starting.py]: Yup

File: src/MainWindow.py (hunk 5 above)
Comment (ivanboika) [reply src/MainWindow.py]: Today?

**Comment by ivanboika** [comment]
I checked about __repr__ and it's optional, if we're talking abt sqlalchemy. I've read it at sqlalchemy docs

**Pull request merged**